Lib/graphlib.py

```diff
@@ -149,15 +152,15 @@ def __bool__(self):
         return self.is_active()
 
     def done(self, *nodes):
-        """Marks a set of nodes returned by "get_ready" as processed.
+        """Marks a set of nodes returned by "get_ready()" as processed.
```

Comment:
I notice that this docstring (and some others) do not follow the common dosctring convention that verbs should be imperative (`Mark`), not indicative.  Would you mind changing them?